migration/i8.2z/postgresql/202106281200_IDEMPIERE-4863.sql

```diff
@@ -0,0 +1,115 @@
+-- Table: adempiere.AD_Wlistbox_Customization
+
+CREATE TABLE adempiere.AD_Wlistbox_Customization
+(
+    AD_Client_ID numeric(10,0) NOT NULL,
+    AD_Org_ID numeric(10,0) NOT NULL,
+    AD_Wlistbox_Customization_ID numeric(10,0) NOT NULL DEFAULT NULL::numeric,
+    WlistboxName varchar(60) NOT NULL,
```

Comment:
since this have to be unique across the application, a longer length should be safer (for e.g 255) here.

Reply:
done